src/js/media-controller.js

```diff
@@ -477,6 +487,28 @@ class MediaController extends MediaContainer {
         );
       },
     };
+
+    // copied from media-chrome-button
+    const keyUpHandler = (e) => {
+      const { key } = e;
+      if (!ButtonPressedKeys.includes(key)) {
+        this.removeEventListener('keyup', keyUpHandler);
+        return;
+      }
+
+      if (!this.hasAttribute('noshortcuts')) {
```

Comment:
should this go into `constants.js` somewhere? MediaUIAttributes doesn't seem correct.
Also, should it be `media-noshortcuts`?

Reply:
I like the compactness of `noshortcuts`, but not super clear to me right off the bat. why not `nokeyboard`?

Reply:
Alternative name: `nohotkeys`